src/lein_monolith/task/each.clj

```diff
@@ -106,6 +119,72 @@
       (->> (map-indexed vector)))))
 
 
+(defn- tee-output-stream
+  "Constructs a proxy of an OutputStream that will write a copy of the bytes
+  given to both A and B."
+  ^OutputStream
+  [^OutputStream out-a ^OutputStream out-b]
+  (proxy [OutputStream] []
```

Comment:
No: `reify` works with protocols and interfaces, `java.io.OutputStream` is a class.